docs/source/guides/upload.mdx

```diff
@@ -195,6 +195,79 @@ notice.
 
 </Tip>
 
+### Regular uploads
+
+The Hugging Face Hub makes it easy to save and version data. However, it has some limitations, especially when it comes
+to updating thousands of times the same file. For instance, you might want to save logs of a training process or user
+feedback on a deployed Space. In those cases, it makes sense to upload the data as a `dataset` on the Hub but it can be
+hard to do it properly. The main reason for that is that you don't want to version every single update of your data -
+which would make the git repository unusable. The [`CommitScheduler`] class offers an interface to solve this problem.
```

Comment:
```suggestion
feedback on a deployed Space. In these cases, uploading the data as a dataset on the Hub makes sense, but it can be hard to do properly. The main reason is that you don't want to version every update of your data because it'll make the git repository unusable. The [`CommitScheduler`] class offers a solution to this problem.
```